packages/core/lib/utils.js

```diff
@@ -1,22 +1,20 @@
-const _ = {
-  flatten: require('lodash/flatten'),
-};
-
 /**
  * Force line returns at specific width. This function is ANSI code friendly and it'll
  * ignore invisible codes during width calculation.
- * @param {string} lines
+ * @param {string} content
  * @param {number} width
  * @return {string}
  */
 exports.breakLines = (content, width) => {
   const regex = new RegExp('(?:(?:\\033[[0-9;]*m)*.?){1,' + width + '}', 'g');
-  return _.flatten(
-    content.split('\n').map((line) => {
+  return content
+    .split('\n')
+    .map((line) => {
```

Comment:
This can be a [`flatMap`](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap)

Reply:
i looked at it earlier and i though so too, didn't know why i didn't do it later